### PR TITLE
Add matplotcheck to conda-forge

### PR DESCRIPTION
**Pull request by lwasser**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example) i'm hoping i did this right but please do say the word if it isn't
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0 **# our build is 0.0.10 -- is there another build somewhere i should set to 0?**
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there *# so far it's just me*

### CODE REVIEW

File: recipes/matplotcheck/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - numpy>=1.14.0
```
Comment (isuruf) [inline recipes/matplotcheck/meta.yaml]: Needs a space before version constraint. (`numpy >=1.14.0)`

Comment (lwasser) [reply recipes/matplotcheck/meta.yaml]: fixed. thank you @isuruf !!!

Comment (lwasser) [reply recipes/matplotcheck/meta.yaml]: @isuruf it looks like CI is happy - all tests passed! what is my next step? wait for a review, etc? many thanks!!

File: recipes/matplotcheck/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
```
Comment (isuruf) [inline recipes/matplotcheck/meta.yaml]: You need to add `python >=3` in `run` too.

Comment (lwasser) [reply recipes/matplotcheck/meta.yaml]: Fixed!!

Comment (lwasser) [reply recipes/matplotcheck/meta.yaml]: ok @isuruf CI is happy again. do you need any other changes ??

**Pull request merged**